modules/accounts/email.nix

```diff
@@ -81,6 +81,14 @@ let
           Note, if set then the system default may still be accepted.
         '';
       };
+
+      fingerprint = mkOption {
+        type = types.nullOr types.str;
+        default = null;
+        description = ''
+          Fingerprint of a trusted TLS certificate.
```

Comment:
The option description should specify the exact format expected. Perhaps the type could verify the format, e.g., using `types.strMatching`?